sdk/boms/azure-core-dependencies/pom.xml

```diff
@@ -0,0 +1,518 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+
+  <groupId>com.azure</groupId>
+  <artifactId>azure-core-dependencies</artifactId>
```

Comment:
Should this be `core-dependencies` or `external-dependencies`? I think `core` can be confusing as it seems like it's related to `azure-core` artifact.